packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -771,8 +771,9 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
         appearance="outline"
         slot="anchor"
         id="people-picker-input"
+        role="combobox"
```

Comment:
@mnickii, should this be done for all the components using this fluentui control?

Reply:
Just the ones using it in a combo box like way, so likely this and the teams channel picker